Kinvey.Core/Realtime/Stream.cs

```diff
@@ -213,9 +214,9 @@ public Stream(string streamName, AbstractClient client = null)
 			}
 
 			// Make KCS request for subscribe access to a user with the given subscribeID
-			success = await RequestSubscribeAccess(subscribeID);
+			var result = await RequestSubscribeAccess(subscribeID);
 
-			if (success)
+			if (result != null)
```

Comment:
When will this be `null`? What about exception cases from the server? Are they thrown up the stack?